opal/mca/common/ucx/common_ucx.c

```diff
@@ -126,6 +127,59 @@ OPAL_DECLSPEC void opal_common_ucx_mca_deregister(void)
     opal_output_close(opal_common_ucx.output);
 }
 
+OPAL_DECLSPEC int opal_common_ucx_priority(ucp_context_h context, int min, int max)
+{
+#if HAVE_DECL_OPEN_MEMSTREAM
+    static const char *transports[] = {
+        "cuda_ipc", "ud_verbs", "rc_verbs", "dc_mlx5"
```

Comment:
no rc_mlx5?

Reply:
IMO, this is a sufficient check, prefer to reduce init time if possible

Reply:
@yosefe We can add "rocm_ipc" to the list as well

Reply:
With ud_verbs in there, aren't we still going to be selecting UCX (over UD) over Libfabric for EFA and/or usnic.